amundsen_application/static/js/interfaces/TableMetadata.ts

```diff
@@ -124,3 +124,32 @@ export interface Watermark {
   partition_value: string;
   watermark_type: string;
 }
+
+export interface LineageItem {
+  badges: string[];
```

Comment:
This should be a list of Badges, not strings. And the implementation will also change in the `LineageList/index.tsx` file accordingly. I'll also add comments there. 

```suggestion
  badges: Badge[];
```

Reply:
Thanks for the catch. I didn't have any sample data and made some assumptions/placeholder code early on and forgot to fix it.